OpenInfraSummit2023/Why-Cloud-Repatriation-is-a-chance-for-OpenStack.md

```diff
@@ -0,0 +1,35 @@
+# Cloud Repatriation as a chance for OpenStack
+
+Track: Public Cloud (or private cloud?)
+
+Authors: Felix Kronlage-Dammers
+
+Technical Level: Intermediate
+
+## Abstract (1000 chars)
+
+Cloud Repatriation is a term that has been buzzing around the cloud industry since
+the paper by a16z and has led to various claims that the age of cloud has its rise.
+With efforts such as the Sovereign Cloud Stack Project more standardiziation of Cloud
+offerings by Cloud Service Providers is anticipated. Such efforts are a big chance
+for OpenStack since it allows that even in cases of Cloud Repatriation OpenStack will
+still be the chosen workhorse, just not in a public but in a private cloud environment.
+This talks wants to revisit the myths of cloud computing and what it really
+has to offer aside from the always-promised easy scaling.
+In the age of Infrastructure-as-Code standardized environments where the IaC
```

Comment:
Hm, this is a really long sentence. I suggest to split it (for easier reading).

> In the age of **standardized** Infrastructure-as-Code environments, OpenStack can offer a lot of benefits – especially if shifted from public cloud to private environments. Backed by shared knowledge and thus fostering operational confidence, the underlying platform can be kept and well-defined standards will assure interoperability.

Or something like this.

Reply:
oh. @itrich! very good suggestion. thanks!